README.md

```diff
@@ -12,21 +12,34 @@ The FastAPI application provides a simple endpoint at the root (`/`) that return
 - [VSCode](https://code.visualstudio.com/)
 - [VSCode Remote - Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
 
-### Steps to Run the Project
+### Steps to Run the Project from Dev Containers
 
 1. Open the project in VSCode.
 2. When prompted, open the project inside the container. (Or press `F1`, type `Remote-Containers: Reopen in Container`).
-3. Once inside the container, open VSCode Terminal and run these commands:
+3. Once inside the container, open VSCode Terminal at the root of the project and run these commands:
 ```
-cd /workspaces/fastapi-example/src
+cd src
 uvicorn main:app --port <PORT>
 ```
 This will start the FastAPI application, and you can access it on port <PORT>
 
-4. Visit `http://localhost:<PORT>`. in your browser to see the current Unix timestamp returned by the FastAPI app.
+4. Visit `http://localhost:<PORT>` in your browser to see the current Unix timestamp returned by the FastAPI app.
 5. Access the API Swagger docs at `http://localhost:<PORT>/docs`.
 
```

Comment:
missing the steps to run the unit tests?

also note that you have to follow the structure as detailed in :

https://github.com/ai-cfia/github-workflows/blob/main/.github/workflows/workflow-lint-test-python.yml

so your code has to pass ruff linting:

`ruff --format=github --target-version=py311 .`

and your tests have to work with the same command-line:

`python -m unittest discover -s tests`

Reply:
done